### DISCUSSION TITLE
Converting ASE Trajectory Files with dpdata

### DISCUSSION THREAD

**Discussion by trevitran** [deepmodeling/deepmd-kit#1305] [Q&A]
Hello,

I am trying to convert my ASE trajectory files with dpdata, however, I am getting the following error message: 
  File "prepare_data.py", line 2, in <module>
    dsys = dpdata.LabeledSystem('cluster1.traj')
  File "/home/trevi.tran/.local/lib/python3.7/site-packages/dpdata/system.py", line 1186, in __init__
    self.from_fmt(file_name, fmt, type_map=type_map, begin= begin, step=step, **kwargs)
  File "/home/trevi.tran/.local/lib/python3.7/site-packages/dpdata/system.py", line 148, in from_fmt
    raise RuntimeError('unknow data format ' + fmt)
RuntimeError: unknow data format traj

I am currently using the following software versions: 
- deepmd-kit 2.0.2
- dpdata 0.2.1
- Python 2.7.5

Any suggestions on how to convert my data? Thank you

**Comment by njzjz** [comment]
Try to add the argument `fmt="ase/structure"`.

**Reply by trevitran** [reply]
Thank you for getting back to me. I am still getting the same error message:
 File "prepare_data.py", line 3, in <module>
    dsys = dpdata.LabeledSystem('cluster1.traj')
  File "/home/trevi.tran/.local/lib/python3.7/site-packages/dpdata/system.py", line 1186, in __init__
    self.from_fmt(file_name, fmt, type_map=type_map, begin= begin, step=step, **kwargs)
  File "/home/trevi.tran/.local/lib/python3.7/site-packages/dpdata/system.py", line 148, in from_fmt
    raise RuntimeError('unknow data format ' + fmt)
RuntimeError: unknow data format traj

Attached is a photo of my script:
<img width="531" alt="Screen Shot 2021-11-27 at 10 48 02 AM" src="https://user-images.githubusercontent.com/90014071/143718694-4a87eb5a-0280-48be-b1fc-dd211aa656bb.png">

The file "cluster1.traj" is my ASE trajectory file.

**Reply by njzjz** [reply]
```py
dsys = dpdata.LabeledSystem("cluster1.traj", fmt="ase/structure")
```

**Reply by trevitran** [reply]
<img width="489" alt="Screen Shot 2021-11-27 at 10 07 42 PM" src="https://user-images.githubusercontent.com/90014071/143731676-f7307ce9-fa42-4eba-a7b9-cf5e1c16cdd6.png">

When I run the updated scripted shown above, I get the following error message:
  File "prepare_data.py", line 2, in <module>
    dsys = dpdata.LabeledSystem("cluster1.traj", fmt="ase/structure")
  File "/home/trevi.tran/.local/lib/python3.7/site-packages/dpdata/system.py", line 1186, in __init__
    self.from_fmt(file_name, fmt, type_map=type_map, begin= begin, step=step, **kwargs)
  File "/home/trevi.tran/.local/lib/python3.7/site-packages/dpdata/system.py", line 148, in from_fmt
    raise RuntimeError('unknow data format ' + fmt)
RuntimeError: unknow data format ase/structure
[trevi.tran@kepler quinoline]$ enw prepare_data.py

**Reply by njzjz** [reply]
Please upgrade dpdata to a newer version.

**Comment by trevitran** [comment]
I upgraded dpdata to a newer version (0.2.4) and received this error message. Is there any other way I can convert my ASE files to be compatible with DeePMD? Thank you.

<img width="959" alt="Screen Shot 2021-12-30 at 11 46 53 AM" src="https://user-images.githubusercontent.com/90014071/147783535-7a0fea2a-1c2d-4d73-b2ed-d7772b2693bc.png">
